databases/litecli/Portfile

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+github.setup        dbcli litecli 1.0.0 v
+revision            0
+
+categories          databases python
+maintainers
+license             BSD
+platforms           darwin
+supported_archs     noarch
+description         CLI for SQLite Databases with auto-completion and syntax highlighting
+long_description    ${description}
+
+homepage            https://litecli.com/
+
+checksums           rmd160  a3fc906e63f010eb92dd2bd5d55cdf7d12525a81 \
+                    sha256  59d652d868ab7525126c7c72b6a39259857f850fdbbe7e2b95c3f4dcc42f1601 \
+                    size    881824
+
+variant python35 conflicts python36 python37 description "Use Python 3.5" {}
+variant python36 conflicts python35 python37 description "Use Python 3.6" {}
+variant python37 conflicts python35 python36 description "Use Python 3.7" {}
```

Comment:
Is there a particular reason to have variants for old python versions instead of just using the latest (py37)?

Reply:
Portfile is based on `pgcli` and `mycli` which support py27, py34, py35, py36 and py37, I only removed variants which were not compatible.

I think keeping backward compatibility with older versions of Python is beyond scope of this PR and maybe should be removed on all dbcli (or even all python-based) tools if maintainers agree to do that.

For know if this tool can work with py35 and py36 as well as other dbcli tools I'd keep it that way.

Reply:
If @xeron agrees to become a maintainer let him make this decision